src/routes/event/index.tsx

```diff
@@ -66,129 +66,145 @@ const Event = ({ eventKey }: Props) => (
+                                <Chip>{`Wk ${eventInfo.week + 1}`}</Chip>
+                              )}
+                            </span>
+                          ),
+                          action: (
+                            <IconButton
+                              href={gcalUrl(eventInfo)}
+                              target="_blank"
+                              rel="noopener"
+                              icon={calendarPlus}
+                            />
+                          ),
+                        },
+                        eventInfo.webcasts.length > 0 && {
+                          icon: video,
+                          title:
+                            'Live Stream' +
+                            (eventInfo.webcasts.length === 1 ? '' : 's'),
+                          href:
+                            eventInfo.webcasts.length === 1
+                              ? eventInfo.webcasts[0]
+                              : undefined,
+                          target: '_blank',
+                          rel: 'noopener',
+                          action:
+                            eventInfo.webcasts.length === 1 ? (
+                              <Icon icon={webcastIcon(eventInfo.webcasts[0])} />
+                            ) : (
+                              eventInfo.webcasts.map(w => (
+                                <IconButton
+                                  key={w}
+                                  icon={webcastIcon(w)}
+                                  href={w}
+                                  target="_blank"
+                                  rel="noopener"
+                                />
+                              ))
+                            ),
+                        },
+                      ]}
+                    />
+                  ) : (
+                    <Spinner />
+                  )}
+                  {nextMatch && (
+                    <MatchCard
+                      key={nextMatch.key}
+                      match={nextMatch}
+                      href={`/events/${eventKey}/matches/${nextMatch.key}`}
+                    />
+                  )}
+                </div>
+              ),
+            },
+            {
+              name: 'Teams',
+              contents: (
+                <div class={style.teamsView}>
+                  {eventStats && schema ? (
+                    <AnalysisTable
+                      teams={eventStats}
+                      schema={schema}
+                      renderTeam={team => (
+                        <a href={`/events/${eventKey}/teams/${team}`}>{team}</a>
+                      )}
+                    />
+                  ) : (
+                    <Spinner />
+                  )}
+                </div>
+              ),
+            },
+            {
+              name: 'Matches',
+              contents:
+                matches === undefined ? (
```

Comment:
All arrays including `[]` are truthy, can you rearrange the cases here back to be similar to how it was, and check for `!matches` instead of `matches === undefined` to display the spinner?